packages/xml-views-validation/api.d.ts

```diff
@@ -20,7 +20,10 @@ export interface OffsetRange {
   end: number;
 }
 
-export type UI5XMLViewIssue = UnknownEnumValueIssue | UseOfDeprecatedClassIssue;
+export type UI5XMLViewIssue =
+  | UnknownEnumValueIssue
+  | UseOfDeprecatedClassIssue
+  | UnknownNamespaceInXmlnsAttributeValueIssue;
```

Comment:
Subjective / Discussion: 

Should this be named `UnknownNamespaceDefinitionIssue`? or something similar without the `AttribueValue` suffix?

Also should we remove the `Issue`  suffix in all these names?

Reply:
If we have other issues about unknown namespace definition (which aren't in the `xmlns:` attribute value) I prefer to have a different type for them.
About the `Issue` suffix I don't really care but it could help to understand what this is with the suffix since we don't use inheritance here.

Reply:
I see this difference / inconsistency:
- `InXmlnsAttributeValue` is described in XML **syntax** information
- `UnknownEnumValue` and `UseOfDeprecatedClassIssue` are described in UI5 VIew semantics.

Reply:
I think if we allow the user to set the severity or turn off validations, it's possible that they won't want this validation in the xmlns attribute but will want it in other places (I don't know where... but the xmlns attribute it doesn't affect the runtime until you try to use it in a tag/attribute name).